<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1342</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="scaleBtn">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>80</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Масштабировать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveBtn">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>180</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Перенести</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotateBtn">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>290</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Повернуть</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadModelBtn">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>570</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Загрузить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>20</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>коэффициенты масштабирования</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>120</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>коэффициенты переноса</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>230</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>коэффициенты поворота</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>50</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>kx</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>50</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ky</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>941</width>
      <height>651</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>50</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>kz</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>150</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>dx</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>150</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>dy</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dyDSB">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>150</y>
      <width>62</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-10000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dxDSB">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>150</y>
      <width>62</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-10000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dzDSB">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>150</y>
      <width>62</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-10000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>150</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>dz</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="kxDSB">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>50</y>
      <width>62</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-10000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>2.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="kyDSB">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>50</y>
      <width>62</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-10000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>2.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="kzDSB">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>50</y>
      <width>62</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-10000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>2.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="scaleAllBtn">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>80</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Масштабировать всё</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveAllBtn">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>180</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Перенести всё</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotateAllBtn">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>290</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Повернуть всё</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>510</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Текущая модель</string>
    </property>
   </widget>
   <widget class="QComboBox" name="modelsCB">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>540</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteModelsBtn">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>610</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить все</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteModelBtn">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>610</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteCameraBtn">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>450</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addCameraBtn">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>410</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cameraCB">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>380</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>350</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Текущая камера</string>
    </property>
   </widget>
   <widget class="QPushButton" name="upBtn">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>370</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>↑</string>
    </property>
   </widget>
   <widget class="QPushButton" name="leftBtn">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>414</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>←</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rigthBtn">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>414</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>→</string>
    </property>
   </widget>
   <widget class="QPushButton" name="downBtn">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>450</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>↓</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>410</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; font-style:italic; color:#c8a8ce;&quot;&gt;*TO-&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ozDSB">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>260</y>
      <width>62</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-10000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>260</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ox</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="oxDSB">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>260</y>
      <width>62</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-10000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="oyDSB">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>260</y>
      <width>62</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-10000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>260</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Oy</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>260</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Oz</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1342</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_info">
    <property name="title">
     <string>информация</string>
    </property>
    <addaction name="menu_author"/>
    <addaction name="menu_task"/>
   </widget>
   <addaction name="menu_info"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menu_author">
   <property name="text">
    <string>автор</string>
   </property>
  </action>
  <action name="menu_task">
   <property name="text">
    <string>задание</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
